enterprise/backend/test/metabase_enterprise/semantic_search/query_test.clj

```diff
@@ -2,116 +2,22 @@
   (:require
    [clojure.test :refer :all]
    [metabase-enterprise.semantic-search.db :as semantic.db]
-   [metabase-enterprise.semantic-search.embedding :as semantic.embedding]
    [metabase-enterprise.semantic-search.index :as semantic.index]
-   [metabase.search.core :as search.core]
-   [metabase.search.ingestion :as search.ingestion]
-   [metabase.test :as mt]
-   [metabase.util.log :as log]
-   [nano-id.core :as nano-id]))
-
-(def ^:private mock-embeddings
```

Comment:
This stuff just moved into a `test-util` namespace so I could use it in the index tests as well

Reply:
thanks!